### PR TITLE
Adding LRP functions to support toast

### PR DESCRIPTION
**Pull request by pmpurifoy**

### CODE REVIEW

File: dev/ToastNotifications/ToastNotificationManager.cpp [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            std::wstring appIdentifier{ RetrieveAppId() };
+            std::wstring storedComActivatorString{ RegisterComActivatorGuidAndAssets(appIdentifier, nullptr) };
+
+            UnRegisterComServer(storedComActivatorString);
+            UnRegisterAppIdentifierFromRegistry(appIdentifier);
+
+            wil::unique_cotaskmem_string processName;
+            THROW_IF_FAILED(GetCurrentProcessPath(processName));
+            auto notificationPlatform{ PushNotificationHelpers::GetNotificationPlatform() };
+
```
Comment (pmpurifoy) [inline dev/ToastNotifications/ToastNotificationManager.cpp:91]: Need to call the remove toast LRP function here.

**Comment by riverar** [comment]
For externals, what is LRP?

**Comment by pmpurifoy** [comment]
> For externals, what is LRP?

LRP is shorthand for the PushNotificationsLongRunningTask which is a process that supports packaged/unpackaged applications using Raw/Toast notifications.

**Comment by pmpurifoy** [comment]
This PR is moving here:

https://github.com/microsoft/WindowsAppSDK/pull/1976

**Pull request closed without merging**